providers/dns/epik/internal/client.go

```diff
@@ -15,6 +15,8 @@ import (
 
 const defaultBaseURL = "https://usersapiv2.epik.com/v2"
 
+const defaultUserAgent = "goacme/lego"
```

Comment:
In #2291, this is `go-acme/lego`, and `ourUserAgent` in `acme/api/internal/sender/useragent.go` is `xenolf-acme/<version>`. I haven't checked, but likely other providers might also set user agents.

Can we just export and use `sender.OurUserAgent`, or does this introduce a circular dependency?

Reply:
This is a special case: https://github.com/go-acme/lego/issues/2268#issuecomment-2394007004

Reply:
About the user `xenolf-acme/<version>`: this is related to Let's Encrypt to ease the detection from their side and for historical reasons it's difficult to change it.

I already think of having a shared user-agent but `sender.OurUserAgent` is difficult to share.
Maybe I should create something inside the `dns01` package.

Reply:
`xenolf-acme/<version>` doesn't contain `go-`, so my point still stands :)

Reply:
This kind of special case is why it's difficult to really share pieces between implementations, there are some unexpected details (ex: `Accept` header, etc.) that break API (yes there is at least one implementation that doesn't support `Accept`... DNS API is a world by itself :smile: ).

Reply:
I don't really want to spread `xenolf-acme/<version>` because it's just an old thing.

Reply:
I will add a comment to explain why it's not the same user agent as the other.

Reply:
LGTM